src/utils/getFilteredPeople.ts

```diff
@@ -0,0 +1,58 @@
+import { Person } from '../types';
+
+export const getFilteredPeople = (
+  people: Person[], filters: URLSearchParams,
+) => {
+  let filteredPeople = [...people];
+  const query = filters.get('query') || '';
+  const sort = filters.get('sort') || '';
+  const order = filters.get('order') || '';
+  const centuries = filters.getAll('centuries') || [];
+  const sex = filters.get('sex') || 'All';
+
+  if (centuries.length) {
+    filteredPeople = filteredPeople.filter(
+      person => centuries.includes(
+        Math.ceil(person.born / 100).toString(),
+      ),
+    );
+  }
+
+  if (sex !== 'All') {
```

Comment:
not critical but `All` is used in many places your code, will be better create `enum` for sort fields